CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.2]
+### Added
+- fix: local.table is object with no attributes on destroy
```

Comment:
```suggestion
- fix: local.table is an object with no attributes on destroy
```

Reply:
that is the 1:1 terraform error message that is fixed..